UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -22,6 +22,8 @@ public class SpriteHandler : MonoBehaviour
 	[SerializeField] private SpriteDataSO PresentSpriteSet;
 	private SpriteDataSO.Frame PresentFrame = null;
 
+	[SerializeField] private bool randomInitialSprite = false;
```

Comment:
add tooltips explaining what it does

Reply:
Done; see https://github.com/unitystation/unitystation/pull/5766/files/682965c38e233ed920f8aecd20360a16b2607e60#diff-ebae75d08489df0a3c75ab87b5b8d5cdd7551b791075ce40301e9bcfbeecfbe8